### PR TITLE
Add Delta Migration table list

### PR DESCRIPTION
**Pull request by brentwpeterson**
## This PR is a:

- New topic

## Summary

Add the table list of from the Delta Migration for the tables that are created in your Magento 1 version

This will add a new page

whatsnew
Added a [new topic](http://devdocs.magento.com/guides/v2.1/migration/migration-migrate-table-list.html) that lists all tables that the migration tool inserts into the M1 database.

### CODE REVIEW

**Comment by magento-cicd2** [comment]
An admin must run tests on this PR before it can be merged.

**Comment by brentwpeterson** [comment]
#mm19in

**Comment by brentwpeterson** [comment]
I added the symlinks for the different versions (2.1->2.3)

File: guides/v2.0/migration/migration-migrate-table-list.md [hunk 1]
```diff
@@ -0,0 +1,78 @@
+---
+group: migration-guide
```
Comment (jeff-matthews) [inline guides/v2.0/migration/migration-migrate-table-list.md]: Only the following meta tags are required:

```
---
title: Table list for M1 delta migration
group: migration-guide
functional_areas:
    - Tools
---
```

Please delete the others.

**Comment by victor-v-rad** [comment]
The tool setup not only tables but **triggers also** and it is already described in [Migrate changes](https://devdocs.magento.com/guides/v2.3/migration/migration-migrate-delta.html) in the first sentence. What I can alternatively propose, is to make popup like this
popup
for the text "m2_cl_*" and "triggers". When someone click text, popup will be shown with several names of tables and triggers. Just as an example. But not to list all table names and triggers because this list can be changed in a future. 

One more proposal is just to mention about deltalog.xml file which contain all these table names in the first paragraph of [Migrate changes](https://devdocs.magento.com/guides/v2.3/migration/migration-migrate-delta.html). Example "The incremental migration tool installs deltalog tables (with prefix m2_cl_*) and triggers (for tracking changes) in the Magento 1 database during the migration of data. Refer to your deltalog.xml.dist file to see the list of all the tables"

**Comment by jeff-matthews** [comment]
Thanks for reviewing @victor-v-rad. We can only use the pop-ups for glossary definitions. Glossary definitions are stored in a separate repo: https://github.com/magento/glossary

We're in the process of updating the README for contributors.

**Comment by devops-devdocs** [comment]
An admin must run tests on this PR before it can be merged.

**Comment by shrielenee** [comment]
Hey @osrecio --- Any updates on this? Are we ready to look at it for approval and merging purposes?

**Comment by osrecio** [comment]
Hi @shrielenee ,

I added a commit to solve the pending request from @jeff-matthews . Now the PR is OK

Could be merged

**Comment by jeff-matthews** [comment]
A few thoughts:

- I don't think we need a new topic for this info, but if we proceed with a new topic, we need a corresponding TOC entry for it.
- I think that @victor-v-rad's proposal is the more efficient solution:

  >One more proposal is just to mention about deltalog.xml file which contain all these table names in the first paragraph of [Migrate changes](https://devdocs.magento.com/guides/v2.3/migration/migration-migrate-delta.html). Example "The incremental migration tool installs deltalog tables (with prefix m2_cl_*) and triggers (for tracking changes) in the Magento 1 database during the migration of data. Refer to your deltalog.xml.dist file to see the list of all the tables"

- If we adopt the suggestion above, then I don't think the following is applicable:
  >and the last proposal is to put it as part of "Data Migration Tool Technical Specification" since it is more about technical nuances, then about user experience. 
 
  I think it's better to mention the `deltalog.xml.dist` file in context in the [Migrate changes](https://devdocs.magento.com/guides/v2.3/migration/migration-migrate-delta.html) topic.

**Comment by shrielenee** [comment]
Hey @jeff-matthews @osrecio @brentwpeterson  --- What's left to do here? Should this still be an applicable PR? ❓

**Comment by jeff-matthews** [comment]
@shrielenee

The only change that is necessary is to add the following sentence to the end of the first paragraph of the [Migrate changes](https://devdocs.magento.com/guides/v2.3/migration/migration-migrate-delta.html) topic:

> Refer to your `deltalog.xml.dist` file to see a list of all the tables.

**Comment by shrielenee** [comment]
> @shrielenee
> 
> The only change that is necessary is to add the following sentence to the end of the first paragraph of the [Migrate changes](https://devdocs.magento.com/guides/v2.3/migration/migration-migrate-delta.html) topic:
> 
> > Refer to your `deltalog.xml.dist` file to see a list of all the tables.

@brentwpeterson Please complete this small change and we'll get it all taken care of!

**Comment by osrecio** [comment]
Hi @shrielenee ,

In order to merge this PR. I can add the small change to [Migrate Changes](https://github.com/magento/devdocs/blob/master/guides/v2.1/migration/migration-migrate-delta.md) topic in this PR.

Adding the small change in the following way:

```
The incremental migration tool installs deltalog tables (with prefix m2_cl_*) and triggers (for tracking changes) in the Magento 1 database during the [migration of data]({{ page.baseurl }}/migration/migration-migrate-data.html). These deltalog tables and triggers are essential to ensuring that you migrate only the changes made in Magento 1 since the last time you migrated data. These changes are:
* data that customers added via storefront (created orders, reviews, changes in customer profiles, etc.)
* all operations with orders in the Magento Admin panel

Refer to your deltalog.xml.dist file to see a list of all the tables.
```

What do you think?

**Comment by victor-v-rad** [comment]
From my side it is Ok

**Comment by shrielenee** [comment]
> The incremental migration tool installs deltalog tables (with prefix m2_cl_*) and triggers (for tracking changes) in the Magento 1 database during the [migration of data]({{ page.baseurl }}/migration/migration-migrate-data.html).
> 
> These deltalog tables and triggers are essential to ensuring that you migrate only the changes made in Magento 1 since the last time you migrated data:
> 
> * Data that customers added via the storefront (created orders, reviews, changes in customer profiles, etc.)
> * All operations with orders in the Magento Admin panel
> 
> Refer to your `deltalog.xml.dist` file to see a list of all the tables.
> 
> {: .bs-callout .bs-callout-info }
> All new or updated entities entered through the Admin panel, like products and categories, are not included in incremental migration and will not be migrated.

**Comment by shrielenee** [comment]
I prefer my sight revision as shown in a previous comment @osrecio. Also ensure that the TOC is showing; add to `_data/toc/migration-guide.yml` file so that it'll display in the left side navigation.

**Comment by shrielenee** [comment]
@osrecio @brentwpeterson 2.0 will be deprecated soon. Info will be disseminated shortly about this. So any change to content needs to be removed from 2.0. Please also review the suggested revisions and give us an update on progress. Thanks!

**Comment by shrielenee** [comment]
Hello! We are archiving this ticket. We have been awaiting an update for quite a while. We're archiving this in an effort to create a quality, community-driven backlog which will allow us to allocate the required attention more easily.
 
If you believe that the reported problem needs further review, please open a new issue. Thank you for your continued collaboration and assistance!

**Pull request closed without merging**